cryptographic_estimators/MQEstimator/MQAlgorithms/hybrid_f5.py

```diff
@@ -124,19 +124,19 @@ def _compute_time_complexity(self, parameters: dict):
 
             sage: from cryptographic_estimators.MQEstimator.MQAlgorithms.hybrid_f5 import HybridF5
             sage: from cryptographic_estimators.MQEstimator.mq_problem import MQProblem
-            sage: H = HybridF5(MQProblem(q=256, n=10, m=10))
-            sage: H._compute_time_complexity({'k':2})
-            36.659592676441406
-
+            sage: H = HybridF5(MQProblem(q=256, n=10, m=10), bit_complexities=False)
+            sage: H.time_complexity(k=2)
+            39.98152077132876
         """
         k = parameters['k']
         n, m, q = self.get_reduced_parameters()
         w = self.linear_algebra_constant()
         degrees = self.degree_of_polynomials()
         E = F5(MQProblem(n=n-k, m=m, q=q), w=w, degrees=degrees)
         E.complexity_type = ComplexityType.ESTIMATE.value
+        E.bit_complexities = False
```

Comment:
the complexity type is by default estimate, so line 136 can be deleteed and the bit_complexity flag should be directly passed to the constructor of F5